x-pack/plugins/apm/server/routes/alerts/chart_preview.ts

```diff
@@ -25,7 +26,10 @@ const alertParamsRt = t.intersection([
     transactionType: t.string,
   }),
   environmentRt,
-  rangeRt,
+  t.type({
+    windowSize: toNumberRt,
+    windowUnit: t.string,
```

Comment:
Let's validate `windowUnit`. It should be one of these:

https://github.com/elastic/kibana/blob/07f49853cd46fd9f4b47f363a0cc3bb091073ec9/x-pack/plugins/triggers_actions_ui/public/application/constants/time_units.ts#L8-L13